router.go

```diff
@@ -186,17 +186,30 @@ func (res CSVResponse) Write(w http.ResponseWriter, r *http.Request) {
 	w.Write(res.CSV)
 }
 
+type LogType string
+
+const (
+	LogTypeInfo  LogType = "INFO"
+	LogTypeWarn  LogType = "WARN"
+	LogTypeError LogType = "ERROR"
+)
+
 // APIResponse response data representation for API
 type APIResponse struct {
 	Error  string      `json:"error,omitempty"`
```

Comment:
can we convert the struct like this ?

```
type APIResponse struct {
	Error  string      `json:"error,omitempty"`
	Warning  string     json:"warning,omitempty"`
	Info  string       `json:"info,omitempty"`
	Status string      `json:"status,omitempty"`
	Data   interface{} `json:"data,omitempty"`
}
```

Reply:
@bidur-phil Even tho we log the errors as info level or warn level, the response is still an error. So I am not sure if this is needed.
cc. @Swechhya @abhyuday